### ISSUE TITLE
UX: Iterate IncLim prototype

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#12341]
## Description

Goal for this ticket: get prototype ready to launch recruiting. (Midpoint review delay could shift that feedback cycle to another ticket.)

### Background

Wes met with VHA stakeholders on Thurs, 2/23 – Lindsey Peace from HEC (Healthcare Eligibility Center) (main stakeholder), as well as SMEs from Enrollment Eilgibility Dept (EED).

We aligned on how some VA benefits should affect the user experience: aid and attendance, housebound, and pension (alone). Changes will be added to [Mural](https://app.mural.co/invitation/mural/vagov6717/1657807571797?sender=u907f83e01e35bb04de6f8139&key=1f5995db-a767-49f9-a6a0-bbd572cdddb9).

## Acceptance Criteria
- [x] Get initial feedback on prototype from Content & IA and incorporate
- [x] Get initial feedback on prototype from VHA stakeholders
- [x] Incorporate feedback
- [x] Support content iteration by SWContent as needed

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [x] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by jilladams** [comment]
@thejordanwood can you confirm points estimate based on what you know today? (Last week we said 8.)

**Comment by thejordanwood** [comment]
Update as of 2/14: Initial feedback came in from Content & IA, but more feedback from stakeholders in needed before we can move forward with this. 

Changing points estimate to a 5 because the Content feedback has been added into the flow.

**Comment by jilladams** [comment]
High priority and should be tagged as such as soon as it's unblocked and pulled back into a sprint. (Untagging for now, as it's making other items hard to manage in the Stretch lane.)

**Comment by thejordanwood** [comment]
Closing this ticket because prototype has been iterated on and is ready for pilot testing. I expect more iterations after pilot testing.
